tests/unit/test_client.py

```diff
@@ -2,12 +2,14 @@
 import json
 import unittest
 from aiohttp import web
+from aiohttp.test_utils import unittest_run_loop
 from textwrap import dedent
 
 from aiosparql.client import (
-    SPARQLClient, SPARQLQueryFormatter, SPARQLRequestFailed)
+    SPARQLClient, SPARQLQueryFormatter, SPARQLRequestFailed
+)
 from aiosparql.syntax import IRI, RDF, Triples
-from aiosparql.test_utils import AioSPARQLTestCase, unittest_run_loop
+from aiosparql.test_utils import AioSPARQLTestCase
```

Comment:
The import is not used anymore. Can you add something to the CI to check for not used imports? I don't know if it's complicated. Otherwise just remove it.